### ISSUE TITLE
Decide on payment methods

### ISSUE THREAD

**Issue by lucymonie** [FAC9/ux-ui-project#19]
We are preparing a prototype with two payment options, credit card and paypal.

It would be great to know if this is what's needed for the users of the system so we can tailor it appropriately.

**Comment by nogainbar** [comment]
yes this is good, do you know what are the implications in terms of percentage payment to paypal? I would like to find the least expensive option, maybe some other method like paypal but for charities? have a look what you can find as I remember looking and paypal has a 3-4% expense on any transaction..

direct pay is obv very good but some people prefer paypal or similar. 

also paypal means that my trainee need to have a paypal account? maybe there is something out there that doesnt require an account? i am not sure..

**Comment by lucymonie** [comment]
Thanks so much for this. I guess to start with, we should set things up to reflect how payments are currently made, and it will be relatively easy to add other methods as you need them. The method of payment that you choose is definitely a business decision for your organisation to make. We're very happy to implement what you decide on!
